app/views/companies/index.html.haml

```diff
@@ -0,0 +1,17 @@
+- if current_user.company.present?
```

Comment:
As per the logic in controller, if company not present it will be redirect. So should we have to check it again here?